src/flight_server.cpp

```diff
@@ -44,6 +45,17 @@ inline static arf::ActionType const kActionSnapDrop {kFlightSnapDrop, "Delete a
 inline static arf::ActionType const kActionTxnBegin {kFlightTxnBegin, "Starts an ACID transaction and returns its ID."};
 inline static arf::ActionType const kActionTxnCommit {kFlightTxnCommit, "Commit a previously started transaction."};
 
+template <typename... args_at>
+void log_message(const char* message, args_at&&... args) {
+    auto now = std::chrono::system_clock::now();
+    std::time_t current_time = std::chrono::system_clock::to_time_t(now);
+    std::string time_str = std::ctime(&current_time);
+    time_str.pop_back();
+    std::printf("%s: ", time_str.c_str());
+    std::printf(message, std::forward<args_at>(args)...);
```

Comment:
It's not a good idea to have multiple `printf` statements, let's merge into one.